index.rst

```diff
@@ -45,12 +42,53 @@ All of this data needs to be associated with this configuration in a way that is
 In this tech-note we discuss the basic organization of configurations and the operational procedures involved in handling CSC configuration.
 The document begins with the most simple case of bringing a CSC up with a specific configuration, and eventually details how to updating and testing newly defined configurations.
 
-Most CSCs handle configuration in the same manner, however, there are a few exception to the rule, notably the Camera CSCs.
-These special cases are addressed in section-appendix-configuration-non-salObj_.
 
-For more technical details about how CSCs handle configuration, including the definition of configuration related topics published by the CSC, see `tstn-017 <https://tstn-017.lsst.io>`__.
+Configuration Interface Summary
+===============================
+
+The standard interface for CSCs is defined in :lse:`209` and is the official reference/requirements document.
+This section provides a very brief overview of the key topics related to configuration that are continually mentioned in the sections below.
+
+The standard configuration related interactions with CSCs happens via two main events (``configurationsAvailable`` and ``configurationsApplied``) and a single parameter as part of the start command (``configurationOverride``).
+When in the STANDBY state, the ``configurationsAvailable`` event is published which a list of non-default (custom) configurations as well as information required to track available configurations (both default and custom) to a specific git commit.
+Under normal operations, site-specific default configurations are automatically loaded.
+However, using the ``configurationOverride`` parameter (explained in detail in the use-cases below), it is possible to specify one of the non-default configurations that is listed in the ``configurationsAvailable`` event.
+Lastly, the ``configurationsApplied`` event is published when the CSC transitions from the STANDBY state to the DISABLED state (which is on the way to the ENABLED state), which contains the information regarding which configuration(s) have been loaded and are in active use by the CSC.
+
+``configurationsAvailable`` Event Description
+---------------------------------------------
+
+The ``configurationsAvailable`` event is a Generic event that is implemented by every configurable CSC.
+It contains the following parameters: 
```

Comment:
Would it make any sense to just refer to the XML, e.g. a link to the appropriate location in the official ts_xml docs? It makes me nervous to duplicate information, and the XML is what users actually have to use.

If the descriptions aren't good enough there, then we need to fix that (independent of whether we keep this text).

Reply:
I looked for this, but the generic events aren't described as a "set" in the ts-xml.lsst.io page. They're essentially just added to every CSC XML. I do think it would be worthwhile to have a docs page about the generics that could be linked from here.
I did add that the definition of the event comes from LSE-209.

Reply:
Created https://jira.lsstcorp.org/browse/DM-33308 and assigned it to Eric... but feel free to re-assign etc.